kernels/ZendEngine3/object.c

```diff
@@ -1194,7 +1194,7 @@ int zephir_create_instance_params(zval *return_value, const zval *class_name, co
 			zval *item;
 			int i = 0;
 
-			if (likely(param_count) <= 10) {
+			if (EXPECTED(param_count <= 10)) {
```

Comment:
Well IN ZE2 this fix - https://github.com/phalcon/zephir/pull/1342/files#diff-a10f1751cb10ff71867e53250b0acb36L1560 fixed an issue anyway